src/site/_includes/pattern-set.njk

```diff
@@ -46,110 +67,71 @@ pageScripts:
   {{ breadcrumbs(crumbs) }}
 </section>
 
-<header class="w-masthead-patterns">
-  <div class="w-masthead-patterns__container w-layout-container--wide">
-    <div class="w-masthead-patterns__left">
-      <h1 class="w-headline--two">
-        {{ patternSuite.title }}
-      </h1>
-      <p>
-        {{ patternSuite.description }}
-      </p>
-    </div>
-    <div class="w-masthead-patterns__right">
-      {% Img
+<header class="hero bg-core-bg">
+  <div class="wrapper">
+    <div class="hero__content--split">
+      <div>
+        <h1 class="text-size-6">{{ patternSuite.title }}</h1>
+        <p>{{ patternSuite.description }}</p>
+      </div>
+      <div class="all-center">
+        {% Img
           height=220,
           width=330,
           src=patternSuite.hero,
           alt="Collection cover image",
           decoding="async",
-          class="w-masthead-path__image"
+          class="hero__content__img"
         %}
+      </div>
     </div>
   </div>
 </header>
 
-<hr>
+<div class="wrapper">
+  <div class="sidebar region flex-align-start flex-dir-rev flex-wrap-no">
+    {% include 'partials/toc-side.njk' %}
 
-<section class="w-layout-container--wide code-patterns-nav--top">
-  <nav class="code-patterns-nav__sets">
-    {% for setId, set in patterns.sets %}
-      {% if set.suite == patternSuite.id %}
-        {% if setId == currentPatternSet.id %}
-          <a class="selected" href="/patterns/{{set.id}}">{{set.title}}</a>
-        {% else %}
-          <a href="/patterns/{{set.id}}">{{set.title}}</a>
-        {% endif %}
-      {% endif %}
-    {% endfor %}
-  </nav>
-  <div class="code-patterns-nav__filter">
-    {# <input type="text" placeholder="filter"/> #}
-  </div>
-</section>
-
-<section class="w-layout-container--wide code-patterns-main">
-  <div>
-    {# Show patterns from current set (if set) or from the whole suite #}
-    {% if currentPatternSet %}
-      <h2 class="w-headline--two">{{ currentPatternSet.title }}</h2>
-      {{ content | safe }}
-      {% for patternId, pattern in patterns.patterns %}
-        {% if pattern.set == currentPatternSet.id %}
-          {{ codePattern(pattern) }}
-        {% endif %}
-      {% endfor %}
+    <div class="prose legacy-rollout">
```

Comment:
The header and article body are misaligned:

![image](https://user-images.githubusercontent.com/1914261/150783144-91999fcf-2e52-4291-9b83-6a25e0755bdb.png)

You can probably add wrapper > prose { // no padding } rule to avoid it.

Reply:
I worry this would effect other uses, so i switched it to flow.